### DISCUSSION TITLE
FE Extremely slow response

### DISCUSSION THREAD

**Discussion by 17096835560** [apache/doris#19377] [A - General / Q&A]
1.2.4.1-rc01。Run Command “show proc '/frontends'\G;”，Returning the result takes over 20 seconds。Server 8-core CPU16G memory，Only deployed on the FE side。

**Comment by LemonLiTree** [comment]
Are other commands working fine?

**Reply by 17096835560** [reply]
Like what

**Reply by LemonLiTree** [reply]
such as show proc backends\G;,select * from table limit 1;

**Comment by yagagagaga** [comment]
Perhaps your current server load is relatively high. You can use the `top` command to check if there are any processes that are consuming significant resources. If everything seems to be okay, perhaps you can restart FE or server.

**Reply by 17096835560** [reply]
top - 15:35:45 up  3:40,  1 user,  load average: 0.00, 0.00, 0.00
Tasks: 238 total,   1 running, 128 sleeping,   0 stopped,   0 zombie
%Cpu(s):  0.1 us,  0.0 sy,  0.0 ni, 99.9 id,  0.0 wa,  0.0 hi,  0.0 si,  0.0 st
KiB Mem :  8141732 total,   120988 free,   708604 used,  7312140 buff/cache
KiB Swap:        0 total,        0 free,        0 used.  7180284 avail Mem 

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND                                                                               
12874 root      20   0   13.8g 564208  50552 S   1.0  6.9   1:02.44 java


FE service restart，It has been restarted, but there is still a problem

**Comment by 17096835560** [comment]
Resolved, requires the server to add hosts resolution

**Comment by 17096835560** [comment]
Resolved, requires the server to add hosts resolution
